src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/EnumGroupAndName.cs

```diff
@@ -28,7 +31,30 @@ public EnumGroupAndName(string group, string name)
             }
 
             Group = group;
-            Name = name;
+            _name = () => name;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EnumGroupAndName structure.
+        /// </summary>
+        /// <param name="group">The group name.</param>
+        /// <param name="name">A func which will return the name.</param>
```

Comment:
Should have said: `func` -> `<see cref="Func{string}"/>`. Similarly `EnumGroupAndName` -> `<see cref="EnumGroupAndName"/>` above (in the original constructor overload as well).